src/test/java/core/basesyntax/PalindromeTest.java

```diff
@@ -6,34 +6,45 @@
 public class PalindromeTest {
     private static final String LETTERS_PALINDROME = "Madam, I'm Adam!";
     private static final String LETTERS_AND_NUMBERS_PALINDROME = "1.) level-1";
+    private static final String LETTERS_AND_NUMBERS_NON_PALINDROME = ".) level-1";
     private static final String NUMBERS_PALINDROME = "123 43 21";
     private static final String NON_PALINDROME = "Nice day!";
 
+
     @Test
     public void isPalindromeWithAllLetters() {
         Palindrome palindrome = new Palindrome();
         boolean actualResult = palindrome.isPalindrome(LETTERS_PALINDROME);
-        Assert.assertTrue("Test faid with text " + LETTERS_PALINDROME, actualResult);
+        Assert.assertTrue("Test failed with text " + LETTERS_PALINDROME, actualResult);
     }
 
     @Test
     public void isPalindromeWithLettersAndNumbers() {
         Palindrome palindrome = new Palindrome();
         boolean actualResult = palindrome.isPalindrome(LETTERS_AND_NUMBERS_PALINDROME);
-        Assert.assertTrue("Test faid with text " + LETTERS_AND_NUMBERS_PALINDROME, actualResult);
+        Assert.assertTrue("Test failed with text " + LETTERS_AND_NUMBERS_PALINDROME,
+                actualResult);
+    }
+
+    @Test
+    public void isPalindromeWithoutNumbersRemove() {
```

Comment:
since you are on this class let's rename method based on the style guide: https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names